include/libtorrent/load_torrent.hpp

```diff
@@ -28,14 +28,20 @@ namespace libtorrent {
 	//   * added to a session via add_torrent() or async_add_torrent()
 	//   * saved as a .torrent_file via write_torrent_file()
 	//   * turned into a magnet link via make_magnet_uri()
+	TORRENT_EXPORT add_torrent_params load_torrent_file(
```

Comment:
I think the documentation (above) should be updated to describe the non-throwing versions. But keep in mind that they may still throw `std::bad_alloc`

Reply:
will do